project.py

```diff
@@ -187,20 +439,22 @@ def video_search(self, video):
         vid.release()
 
 
-image_extension_list = ["jpg", "gif", "png"]
-video_extension_list = ["mkv", "wmv", "avi", "mp4"]
-
+video_extension_list = ("mkv", "wmv", "avi", "mp4")
 
-def main(inputFile):
-    board = BoardSearcher()
 
-    if(any(inputFile[-3:] == i for i in video_extension_list)):
-        board.video_search(inputFile)
-    elif (any(inputFile[-3:] == i for i in image_extension_list)):
-        board.image_search(inputFile)
-    else:
-        print "Unrecognized file format"
-    cv.destroyAllWindows()
+def main(input_file):
+    board = BoardSearcher(n_slides=0, frame_counter=0,
```

Comment:
@Foxiy you have quite a lot of options here. I believe they should be all configurable from the command line. The sysadmin in you will surely agree with me =)